src/Home.jsx

```diff
@@ -1,45 +1,48 @@
-import Header from './Header'
-import Nav from './Nav'
+
 import { useEffect, useState } from 'react'
 import { getArticles } from './api'
+import { Link } from "react-router-dom"
 
 
-function Home(){
+function Home({setCurrentArticle}){
     const [allArticles, setAllArticles] = useState([])
-
     useEffect(() => {
         getArticles().then(({articles}) => {
             setAllArticles(articles)
         })
     }, [])
 
+    function HandleArticleClick(e){
+        setCurrentArticle(e.target.id)
+    }
 
-    return <div>
-    <Header/>
-    <Nav/>
+    return <>
     <main className='home-main'>
     {allArticles.map((article, index) => {
         if (index < 6) {
             const day = Date(article.created_at)
-            return <article key={index + 'h-a'} className='home-art'>
-                <img src={article.article_img_url} alt="" />
-            <h3>{article.title}</h3>
-            <h5>{article.topic}</h5>
-            <h5>{day.split('GMT')[0]}</h5>
+            return <Link className='home-art' to= {`/single-article`} key={index + 'h-a'} onClick={(e)=> {HandleArticleClick(e)}}><article  >
```

Comment:
The key needs to be unique, using index could potentially cause some problems. Better to use the article.article_id as the id is definitely unique
Also as the feedback above mentioned, you probably want to show different url to the user when showing different articles, so need to change the link to url as well